src/js/services.js

```diff
@@ -1,47 +1,59 @@
 import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
+import promise from 'es6-promise';
+import fetch from 'isomorphic-fetch';
+import 'babel-polyfill';
+import _ from 'lodash';
+
+const monthNames = ["Jan", "Feb", "March", "April", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+let timelineArray;
 
 const renderedTimeline = (rawData) => {		
-	const monthNames = ["Jan", "Feb", "March", "April", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
-	let timelineArray = [];
-	for(let i in rawData) {
-		let tweetObj = rawData[i];
-		let date = new Date(tweetObj.createdAt);
+	timelineArray = [];
+	timelineArray = _.map(rawData, function(tweetObj, index) {
+		return renderTweetObj(tweetObj, index);
```

Comment:
nice, i was hoping you'd use _.map. Are you sure this is the most concise way to use it? (I actually don't know for sure but I feel like it isn't) Recall renderTweetObj is itself a function.